ci/cleanup_do_ci.sh

```diff
@@ -0,0 +1,10 @@
+rm -rf \
```

Comment:
non-blocking given urgency. I propose in our follow up thread we also update our make targets to have some subsection of this

Reply:
I think they are all overwritten given our use of `cp -f` but I mainly added this to have for local dev. I also added the files to `.gitignore` but I think this still has value